src/renderer.js

```diff
@@ -159,25 +159,36 @@ document.addEventListener("DOMContentLoaded", function () {
   var calendar = new FullCalendar.Calendar(calendarEl, {
     initialView: "dayGridMonth",
     dateClick: function (info) {
-      // Use the load dates for the current and day after the current day here
-      monthNumber = info.dateStr[5] + info.dateStr[6];
-      dayNumber = info.dateStr[8] + info.dateStr[9];
-      loadCurrentDayTasks(dayNumber, monthNumber);
+      const clickedDate = new Date(info.dateStr);
+      const clickedDay = clickedDate.getDate();
+      const clickedMonth = clickedDate.getMonth() + 1;
+
+      const tomorrowDay = clickedDate.getDate();
```

Comment:
For simplicity sake we retrieve the tomorrow date before calling the loadTomorrowTasks function in here.